zadanie.py

```diff
@@ -0,0 +1,30 @@
+foo = []
+
+from .models import Expense
+from collections import namedtuple
```

Comment:
change import order, package ones `.models` should be after built-ins or use isort for automatic job